internal/rpc/rpcserver/server.go

```diff
@@ -61,9 +61,9 @@ import (
 
 // Public API version constants
 const (
-	semverString = "7.12.0"
+	semverString = "7.13.0"
 	semverMajor  = 7
-	semverMinor  = 12
+	semverMinor  = 13
```

Comment:
When you rebase, you'll find it's already at 7.13, so you may wish to bump (or just leave it since it is recently bumped on master)

Reply:
Will leave as it is then.